README.md

```diff
@@ -17,19 +17,25 @@ Web app for finding hygiene resources in Portland, OR
     $ virtualenv env --no-site-packages
     $ source env/bin/activate
     ```
-6. After you’ve created and activated a virtual environment, install Django. The recommended way to install Django is with ```pip```:
+6. After you’ve created and activated a virtual environment, install Django and Django REST framework. The recommended way to install Django is with ```pip```:
     ```sh
     $ [sudo] pip install Django
+    $ [sudo] pip install djangorestframework
```

Comment:
i also had to install something called "Pillow". not sure if that's just because of my lack-luster python environment but it may be something we want to add here, too.

Reply:
alright, that was my next question!